src/platform/mhv/secondary-nav/components/MhvSecondaryNavMenu.jsx

```diff
@@ -7,12 +7,43 @@ import MhvSecondaryNavItem from './MhvSecondaryNavItem';
  * MHV secondary navigation bar.
  * LIMITATIONS: The logic to set an item to active is based on a first match approach
  * looking through the list of items in reverse order. As such, '/my-health' home
- * page is expected to be the first item in the list.
+ * page is expected to be the first item in the item list. The first item in the list
+ * is expected to be the header item.
  *
- * @param {SecondaryNavItem[]} items the list of items to display in the navigation bar
+ * @property {Object[]} items the list of items to display in the navigation bar
  * @returns the navigation bar
  */
 const MhvSecondaryNavMenu = ({ items }) => {
+  /**
+   * Strip the trailing slash in a path if it exists.
+   * @param {String} path the path
+   * @returns the path without a trailing slash
+   */
+  const stripTrailingSlash = path => path.replace(/\/$/, '');
+
+  /**
+   * Find which navigation item needs to be set to active, if any. An item should be active
+   * when the URL pathname starts with the app's root URL, or the href matches the current
+   * URL pathname.
+   * @param secNavItems the list of navigation items
+   * @returns the item to be set as active, or null if none found
+   */
+  const findActiveItem = (secNavItems = items) => {
+    // Perform a reverse find to match which nav link we are on, so we match on the home page last
+    return [...secNavItems] // Clone the array, so the original stays the same
+      .reverse()
```

Comment:
I take it `toReversed()` isn't in our babel polyfills yet… interesting that the non-mutating method is so new